swift/README.md

```diff
@@ -7,5 +7,11 @@ and Cocoapods
 `pod 'FlatBuffers'`
 
 ### Notes
+
 1- To report any error please use the main repository.
-2- The package 0.4.0 will break the generated code. You can download the [binary here](https://github.com/google/flatbuffers/actions) and select the latest push to master
\ No newline at end of file
+
+2- `0.6.0` deprecates `add(condition:bool)` for `add(element:bool)`. You can download the [binary here](https://github.com/google/flatbuffers/actions) and select the latest push to master
+
+### Contribute
+
+1- Always run `swift test --generate-linuxmain` whenever new test functions are added or removed
```

Comment:
@aardappel Would a small swift specific `contribute` section be fine here?

Reply:
Yes, don't see why not.